resources/resources.go

```diff
@@ -1,15 +1,9 @@
 package resources
 
-import (
-	"encoding/json"
-	"fmt"
-	"net/http"
-)
-
 const (
-	SPECTRUM_SCALE     Backend = "spectrum-scale"
-	SPECTRUM_SCALE_NFS Backend = "spectrum-scale-nfs"
-	SOFTLAYER_NFS      Backend = "softlayer-nfs"
+	SpectrumScale    Backend = "spectrum-scale"
```

Comment:
Any reason for converting constants to camel case ?. Ideally we'd like to distinguish between constants and regular variables